akka-docs/src/main/paradox/stream/operators/Compression/deflate.md

```diff
@@ -0,0 +1,28 @@
+# Compression.deflate
+
+Creates a flow that deflate-compresses a stream of ByteStrings. Use the overload with the `level: Int, nowrap: Boolean` parameters to control the compression level and compatinilty with GZip.  
```

Comment:
`@scala`/`@java` parameter syntaxes, also since this is sentence pulled into the toc I think it would be better to talk about the parameters down in the description than up here.

also "compatinilty"

Reply:
Agreed, I think that first line is sufficient for the 1-liner description and the rest can come below

Reply:
TIL.